### ISSUE TITLE
Search Box Doesn't Have Keyboard Support

### ISSUE THREAD

**Issue by mt-xing** [cornell-dti/samwise#335]
When searching for a class, neither the tab key nor up/down arrows allow you to select search results.
This probably violates some accessibility guideline.

**Comment by SamChou19815** [comment]
This is both a problem on the staging site and production site.
However, now we completely control the `SearchBox` component so we can actually solve this problem!
